lib/sample_search_api/sample_search_apiImpl.py

```diff
@@ -1,31 +1,97 @@
+# -*- coding: utf-8 -*-
 #BEGIN_HEADER
 import logging
 import os
+import json
 
 from installed_clients.KBaseReportClient import KBaseReport
+from installed_clients.SampleServiceClient import SampleService
+from utils.filter_samples import SampleFilterer
 #END_HEADER
-#BEGIN_CLASS_HEADER
-#END_CLASS_HEADER
-#BEGIN_CONSTRUCTOR
+
+
+class sample_search_api:
+    '''
+    Module Name:
+    sample_search_api
+
+    Module Description:
+    A KBase module: sample_search_api
+    '''
+
+    ######## WARNING FOR GEVENT USERS ####### noqa
+    # Since asynchronous IO can lead to methods - even the same method -
+    # interrupting each other, you must be *very* careful when using global
+    # state. A method could easily clobber the state set by another while
+    # the latter method is running.
+    ######################################### noqa
+    VERSION = "0.0.1"
+    GIT_URL = ""
+    GIT_COMMIT_HASH = "35ee1b1a52d7e858fe39160d50ed3d43045381a9"
+
+    #BEGIN_CLASS_HEADER
+    #END_CLASS_HEADER
+
+    # config contains contents of config file in a hash or None if it couldn't
+    # be found
+    def __init__(self, config):
+        #BEGIN_CONSTRUCTOR
         self.callback_url = os.environ['SDK_CALLBACK_URL']
+        # self.re_api_url = config.get('re-api-url', config.get('kbase-endpoint') + '/relation_engine_api')
```

Comment:
is this config var used elsewhere? or should it be removed from the config file?